### ISSUE TITLE
Staging Review finding: Duplicate "Call us" section links in sidebar

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#90540]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vfep-codereviewers
**Product:** Resources for schools
**Feature:** Convert Legacy Markdown SCO page into a React App

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

The links under the `Call Us` section display both the name of the entity that can be called, and its phone number. For each entity, both the name of the entity and the phone number are separate links to the same phone number.  Some are using the `va-telephone` component, some are not.

<details><summary>Link, screenshot or steps to recreate</summary>

![Screenshot 2024-08-12 at 11 05 09 AM](https://github.com/user-attachments/assets/db5c4b03-6cc3-42a5-a8dd-0e783fdb6ddc)

</details>

## Recommended action

Provide only one phone number link for each entity, and ensure that it uses the [`va-telephone`](https://design.va.gov/components/telephone) component. I recommend displaying the organization name as plain text and linking the digits of the phone number. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Telephone ](https://design.va.gov/components/telephone)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by mghisilieri** [comment]
Work for this has been done as part of the following PR: https://github.com/department-of-veterans-affairs/vets-website/pull/31502

**Comment by gforbes4** [comment]
Issue has been completed.
